### PR TITLE
Add io.github.kriptolix.Poliedros

### PR DESCRIPTION
**Pull request by kriptolix**
### Please confirm your submission meets all the criteria

- [x] Please describe the application briefly. 

Poliedros is a multi-type dice roller.

- [x] The domain used for the application ID is https://github.com/kriptolix/Poliedros and the [application id guidelines][appid] are followed.
- [x] I have read and followed all the [Submission requirements][reqs] and the [Submission guide][reqs2].
- [x] I have [built][build] and tested the submission locally.
- [x] I am an author/developer/upstream contributor to the project. If not, I contacted upstream developers about this submission. **Link:**

[appid-domain]: https://docs.flathub.org/docs/for-app-authors/requirements/#control-over-domain-or-repository
[appid]: https://docs.flathub.org/docs/for-app-authors/requirements#application-id
[reqs]: https://docs.flathub.org/docs/for-app-authors/requirements
[reqs2]: https://docs.flathub.org/docs/for-app-authors/submission
[build]: https://docs.flathub.org/docs/for-app-authors/submission/#before-submission

### CODE REVIEW

**Comment by hfiguiere** [comment]
>  Please describe the application briefly. 

this is missing

File: io.github.kriptolix.Poliedros.desktop.in
Comment (hfiguiere) [inline io.github.kriptolix.Poliedros.desktop.in:1]: this should be upstream

File: io.github.kriptolix.Poliedros.metainfo.xml.in
Comment (hfiguiere) [inline io.github.kriptolix.Poliedros.metainfo.xml.in:1]: this should be upstream

Comment (hfiguiere) [reply io.github.kriptolix.Poliedros.metainfo.xml.in:1]: it's not even used.

File: io.github.kriptolix.Poliedros.desktop.in
Comment (hfiguiere) [reply io.github.kriptolix.Poliedros.desktop.in:1]: it's not even used.

File: io.github.kriptolix.Poliedros.metainfo.xml.in [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+
+  <!-- Applications should set a brand color in both light and dark variants like so -->
+  <branding>
+    <color type="primary" scheme_preference="light">#ff00ff</color>
+    <color type="primary" scheme_preference="dark">#993d3d</color>
+  </branding>
+
+  <screenshots>
+    <screenshot type="default">
+      <image>https://github.com/kriptolix/Poliedros/blob/master/data/screenshots/dark-basic-sidebar.png</image>
```
Comment (hfiguiere) [inline io.github.kriptolix.Poliedros.metainfo.xml.in:42]: never reference from the tip of a branch. It's guaranteed to break.

Comment (hfiguiere) [reply io.github.kriptolix.Poliedros.metainfo.xml.in:42]: also this is not a URL to an image, but to the github page.

File: io.github.kriptolix.Poliedros.metainfo.xml.in [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+    <screenshot>
+      <image>https://github.com/kriptolix/Poliedros/blob/master/data/screenshots/dark-command-sidebar.png</image>
+      <caption>Command Mode.</caption>
+    </screenshot>    
+  </screenshots>
+
+  <releases>
+    <release version="1.0" date="2024-01-18">
+      <url type="details">https://example.org/changelog.html#version_1.0.1</url>
+      <description translate="no">
```
Comment (hfiguiere) [inline io.github.kriptolix.Poliedros.metainfo.xml.in:62]:
```suggestion
      <description>
```

File: io.github.kriptolix.Poliedros.svg
Comment (hfiguiere) [inline io.github.kriptolix.Poliedros.svg:1]: this should be upstream

and it's never used here.

File: io.github.kriptolix.Poliedros.json [hunk 3]
```diff
@@ -0,0 +1,39 @@
+{
+    "id" : "io.github.kriptolix.Poliedros",
+    "runtime" : "org.gnome.Platform",
+    "runtime-version" : "47",
+    "sdk" : "org.gnome.Sdk",
+    "command" : "poliedros",
+    "finish-args" : [
+        "--share=network",
```
Comment (hfiguiere) [inline io.github.kriptolix.Poliedros.json:8]: why does this need network access?

File: io.github.kriptolix.Poliedros.metainfo.xml.in (hunk 1 above)
Comment (kriptolix) [reply io.github.kriptolix.Poliedros.metainfo.xml.in:42]: This is the first time I've made a submission to flathub, this is my first flatpak. There are a lot of things that I'm still understanding how works, so I apologize. What would be the correct way to reference a screenshot image?

File: io.github.kriptolix.Poliedros.desktop.in
Comment (kriptolix) [reply io.github.kriptolix.Poliedros.desktop.in:1]: Removed.

File: io.github.kriptolix.Poliedros.metainfo.xml.in
Comment (kriptolix) [reply io.github.kriptolix.Poliedros.metainfo.xml.in:1]: removed

File: io.github.kriptolix.Poliedros.svg
Comment (kriptolix) [reply io.github.kriptolix.Poliedros.svg:1]: Removed.

File: io.github.kriptolix.Poliedros.json (hunk 3 above)
Comment (kriptolix) [reply io.github.kriptolix.Poliedros.json:8]: No need, I just forgot to remove. I'm going to remove.

**Comment by kriptolix** [comment]
When I went to make the required changes I ended up doing a wrong sync, so I'm going to close this PR, fix the problems and then submit another one to avoid all the mess of commits that ended up going into this PR.

**Comment by kriptolix** [comment]
When I went to make the required changes I ended up doing a wrong sync, so I'm going to close this PR, fix the problems and then submit another one to avoid all the mess of commits that ended up going into this PR.

**Pull request closed without merging**